docs/user/openstack/privileges.md

```diff
@@ -0,0 +1,51 @@
+# Required Privileges
+
+In order to install an OpenShift cluster on OpenStack successfully, the user provided to the installer needs certian privileges. The easiest way to achieve this level of permission and ensure success is to install with a user who has administrative privileges.
+
+If the provided user has admin privileges, no action is required and you can skip to the next step. Otherwise, the rest of this document can be used as a resource to create a user with more fine-grained privileges.
+
+
+## Default Install Privileges
+
+To install a basic, externally routable, OpenShift cluster using only whats offered in the installer prompts, your user needs this minimum subset of permissions.
+
+### Networking
+
+| Resource | Permissions |
+|    ---   |     ---     |
+| Network           | All |
```

Comment:
probably all permissions is not what you need here either, for instance, you just need to be able to create a network and a router and then connect them to the external/provider network, but you don't need to permissions to create the external network

Reply:
Would calling this "Private Network" be more clear?